meadServer/client/src/App.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
`fetch('/api/batchData/' + this.selected)`

You can interpolate strings: ` `/api/batchData/${this.selected}` `